store/index.ts

```diff
@@ -69,3 +69,23 @@ export const mutations: MutationTree<RootState> = {
     Vue.set(state.socket, 'reconnectError', true);
   }
 };
+
+export const actions: ActionTree<RootState, RootState> = {
+  async reset({ dispatch }, { clearCritical }: { clearCritical: boolean }) {
+    const promises = [];
+    promises.push(dispatch('backdrop/clearBackdrop', { root: true }));
+    promises.push(dispatch('deviceProfile/clearDeviceProfile', { root: true }));
+    promises.push(dispatch('displayPreferences/resetState', { root: true }));
+    promises.push(dispatch('homeSection/clearHomeSection', { root: true }));
+    promises.push(dispatch('page/clearPage', { root: true }));
+    promises.push(dispatch('playbackManager/stop', { root: true }));
+    if (clearCritical)
```

Comment:
For a one line thing it's good to do it this way, but in this huge block I would use the brace style

Reply:
Wdyn? The if only applies to the server clear. Or I misunderstand you :p

Reply:
I meant to scope the if block in braces:

if (whatever) {
//Whatever
}

Reply:
Done